src/pages/admin/materiaalaanvragen/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { GetServerSidePropsResult, NextPage } from 'next';
+import { GetServerSidePropsContext } from 'next/types';
+import React, { ComponentType } from 'react';
+
+import { AdminLayout } from '@admin/layouts';
+import { withAuth } from '@auth/wrappers/with-auth';
+import PermissionsCheck from '@shared/components/PermissionsCheck/PermissionsCheck';
+import { getDefaultServerSideProps } from '@shared/helpers/get-default-server-side-props';
+import { renderOgTags } from '@shared/helpers/render-og-tags';
+import useTranslation from '@shared/hooks/use-translation/use-translation';
+import { DefaultSeoInfo } from '@shared/types/seo';
+
+const AdminMaterialRequests: NextPage<DefaultSeoInfo> = ({ url }) => {
+	const { tText } = useTranslation();
+
+	const renderPageContent = () => {
+		return (
+			<AdminLayout
+				pageTitle={tText(
+					'pages/admin/mijn-materiaalaanvragen/index___mijn-materiaalaanvragen'
+				)}
+			>
+				<AdminLayout.Content />
+			</AdminLayout>
+		);
+	};
+	return (
+		<>
+			{renderOgTags(
+				tText('pages/admin/mijn-materiaalaanvragen/index___mijn-materiaalaanvragen'),
+				tText(
+					'pages/admin/mijn-materiaalaanvragen/index___mijn-materiaalaanvragen-meta-omschrijving'
+				),
+				url
+			)}
+
+			<PermissionsCheck allPermissions={[]}>{renderPageContent()}</PermissionsCheck>
```

Comment:
You can create a permission and give it the same name as the backend:
https://github.com/viaacode/hetarchief-proxy/blob/release/archief2-fase2/src/modules/users/types.ts#L53

you can add the permissions here:
https://github.com/viaacode/hetarchief-client/blob/master/src/modules/account/const/index.tsx#L90

The 2 Permission enum objects in back and frontend should stay in sync at all time